python-data-version-control/src/prepare.py

```diff
@@ -0,0 +1,43 @@
+from pathlib import Path
+import pandas as pd
+
+folders_to_labels = {"n03445777": "golf ball", "n03888257": "parachute"}
+
+
+def get_filelist_and_labels(source_path, target_folders):
+    images = []
+    labels = []
+    for image_path in source_path.rglob("*/*.JPEG"):
+        filename = image_path.absolute()
+        folder = image_path.parts[-2]
```

Comment:
Instead of subscribing the `.parts` list, it might be more readable to use `.parent.name`:

```suggestion
        folder = image_path.parent.name
```